src/app/Processor/ProcessorEdit/ConfigurationEdit/ConfigurationEdit.tsx

```diff
@@ -106,40 +85,6 @@ const ConfigurationEdit = (props: ConfigurationEditProps): JSX.Element => {
     }
   }, [props]);
 
-  useEffect(() => {
-    if (type) {
-      setSchemaLoading(true);
-      setSchema(undefined);
-      getSchema(type, configType)
```

Comment:
I've moved the retrieval of the schema up one level inside `ProcessorEdit` because I needed it there too.